website/src/components/mdx/tabs/tab-groups.tsx

```diff
@@ -26,7 +26,9 @@ const TabGroupContext = createContext<TabGroupContextShape>({
 
 const getLocalStorageKey = (groupId: string) => `tab-${groupId}`;
 
-export const TabGroupProvider: FC<PropsWithChildren> = ({ children }) => {
+export const TabGroupProvider: FC<PropsWithChildren<unknown>> = ({
```

Comment:
same here

Reply:
same than above, there were 4 or 5 places with the same ... once we update to v18 types, we will remove them.